### PR TITLE
Space Travelers' Hub PR

### PR DESCRIPTION
**Pull request by kkasaija**
### Milestone Reached

- [x] Used React documentation.
- [x] Used React components.
- [x] Used React props.
- [x] Used React Router.
- [x] Connected React and Redux.
- [x] Handled events in a React app.
- [x] Wrote unit tests with React Testing Library and Jest.
- [x] Used styles in a React app.
- [x] Used React life cycle methods.
- [x] Applied React best practices and language style guides in code.
- [x] Used store, actions, and reducers in React.

### CODE REVIEW

File: src/redux/missions/missions.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      return rejectWithValue(error.response.data);
+    }
+  });
+const missionSlice = createSlice({
+  name: 'missions',
+  initialState: {
+    missionList: [],
+    status: null,
+    error: null,
+  },
```
Comment (DammyShittu) [inline src/redux/missions/missions.js:20]: - [ ] You are required to filter the results from the API after fetching the missions, before dispatching to the store. Right now only the rockets values are filtered. Kindly filter the missions as well and save ONLY these values in the store: `mission_id`, `mission_name`, `description`.

  irr

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Kasaija-Kenneth @eerapheal,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @eerapheal , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/missions/missions.js (hunk 1 above)
Comment (kkasaija) [reply src/redux/missions/missions.js:20]: Noted with thanks

**Comment by kkasaija** [comment]
Much appreciated 🙏🙏🙏🙏🙏🙏

**Pull request merged**